### DISCUSSION TITLE
Cannot add games from GOG "cannot find config.json" error

### DISCUSSION THREAD

**Discussion by Solstar** [PhilipK/BoilR#286] [Q&A]
on pc dekstop, no steam deck, tried adding origin uplay and gog games, origin and uplay have been found with no issues, while on gog keeps saying that is impossible to find c:\program files x86\gog galaxy\config. json" file. there is none in that gog galaxy folder, so how do i create one

**Comment by Solstar** [comment]
anyone?

**Comment by zjDrummond** [comment]
Having the same issue. It was working, but something broke it somehow. Not sure what happened.

**Reply by PhilipK** [reply]
Is this on windows as well?

**Comment by Solstar** [comment]
Yes it's on windows
--
Inviato da Libero Mail

**Comment by adriana-olszak** [comment]
Hey, I have the same issue after installing GOG via Moraroy's NonSteamLaunchers. Used a workaround and passed in the installation path of GoG Galaxy/Games as a config for Mini Galaxy and it has worked :)

**Comment by IvarLJ** [comment]
Same as Adriana - have a SteamDeck and installed a couple of launchers using _NonSteamLaunchers_. So far all games I had installed through the Epic Launcher were identified, added and launch from game mode without major issues (small niggles with some of the Epic overlays when launching a game).

But when it comes to GoG, no luck. I pointed BoilR to the config.json file from GoG, but still no games are picked up.

I'll try passing the same config file though the Mini Galaxy config as Adriana suggests to see if that does the trick.
